experimental/framework/tinylicious-client/src/index.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FluidTinylicious } from "./FluidStaticTinylicious";
+export * from "./tinyliciousService";
+
+export { InsecureTinyliciousUrlResolver } from "@fluidframework/tinylicious-driver";
+
+// eslint-disable-next-line import/no-default-export
+export default FluidTinylicious;
```

Comment:
I'm thinking I like just Tinylicious? or TinyliciousClient? 

The reason I'm not a fan of Fluid* is because you're already importing it from a package called @fluidframework/... so it's a bit needless imo.

Reply:
Changed to TinyliciousClient. Keeping it just Tinylicious may make it difficult to distinguish between the Tinylicious service package and the value here